cmd/skuba/addon/upgrade.go

```diff
@@ -60,7 +66,11 @@ func newUpgradeApplyCmd() *cobra.Command {
 		Use:   "apply",
 		Short: "Apply addon upgrade",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := addons.Apply(); err != nil {
+			clientSet, err := kubernetes.GetAdminClientSet()
+			if err != nil {
+				klog.Errorf("Can not retrieve the ClientSet from kubernetes: %v", err)
+			}
```

Comment:
[1]

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
I would rather to change remove.go to make it consistent.